RecoLocalMuon/GEMRecHit/test/TestGEMRecHitAnalyzer.cc

```diff
@@ -72,8 +72,7 @@
 #include <DataFormats/CSCRecHit/interface/CSCSegmentCollection.h>
 #include <Geometry/CommonDetUnit/interface/GeomDet.h>
 #include <Geometry/Records/interface/MuonGeometryRecord.h>
-#include <Geometry/CommonTopologies/interface/RectangularStripTopology.h>
-#include <Geometry/CommonTopologies/interface/TrapezoidalStripTopology.h>
+#include <Geometry/CommonTopologies/interface/GEMStripTopology.h>
```

Comment:
The replacement is in line with the overall migration performed in this PR,
On the other hand, this include doesn't even seem to be needed at all in this test analyzer (as they were not need also the previous two which are being replaced).
Not really an issue, anyhow...